pkg/render/manager.go

```diff
@@ -850,6 +913,11 @@ func managerClusterRole(managedCluster bool, kubernetesProvider operatorv1.Provi
 				},
 				Verbs: []string{"list"},
 			},
+			{
```

Comment:
Could you add a comment in which contexts these permissions are used? Just so we keep better track as to why permissions are introduced.